mypy.ini

```diff
@@ -9,21 +9,21 @@ warn_unused_ignores = True
 warn_redundant_casts = True
 
 # We ignore errors everywhere to gradually add type annotations
-
-[mypy-lm_eval.*]
-ignore_errors = True
-
-[mypy-lm_eval.api.*]
-ignore_errors = True
-
-[mypy-lm_eval.prompts.*]
-ignore_errors = True
-
-[mypy-lm_eval.models.*]
-ignore_errors = True
-
-[mypy-scripts.*]
-ignore_errors = True
-
-[mypy-main]
-ignore_errors = True
+;
+;[mypy-lm_eval.*]
```

Comment:
what do these semicolons do here? are they just commenting these lines out?

Sorry, couldn't immediately find docs on this.

Reply:
yeah. The IDE automatically used them. Apparently they are equivalent but semi-colons have less support or something. Changed to hash.